generators/qunit/templates/test/testsuite.qunit.js

```diff
@@ -0,0 +1,11 @@
+window.suite = function () {
+    "use strict";
+
+    var oSuite = new parent.jsUnitTestSuite(),
+        sContextPath = location.pathname.substring(0, location.pathname.lastIndexOf("/") + 1);
+        
+    oSuite.addTestPage(sContextPath + "unit/unitTests.qunit.html");
+    <% if (typeof(opa5Journeys) !== "undefined") { %>oSuite.addTestPage(sContextPath + "integration/opaTests.qunit.html");
```

Comment:
Would that make sense to apply the journey splitting pattern ?

Reply:
We can discuss that for the opa-generator.